src/core/browser.js

```diff
@@ -15,7 +15,10 @@ export const getBrowser = () => {
 
     const name = browserInfo.getBrowserName();
     const version = browserInfo.getBrowserVersion();
-    const isSilk = name === "Amazon Silk";
+    const isKindleWebView = new RegExp(
+        /KFOT|KFTT|KFJWI|KFJWA|KFOTE|KFSOWI|KFTHWI|KFTHWA|KFAPWI|KFAPWA|WFJWAE|KFSAWA|KFSAWI|KFASWI|KFARWI|KFFOWI|KFGIWI|KFMEWI/g,
```

Comment:
🤔

Reply:
ah android build refs or something?

Reply:
Yeah list supplied by apps. Not sure any other ways to narrow it down exist. Was hoping could add " Build" to the end but that may or may not be present.